jest/verifyComponent.tsx

```diff
@@ -21,7 +21,7 @@ export function verifyComponent(
   Component: React.ComponentType<any>,
   {modelFn, props}: {modelFn?: Function; props: object}
 ) {
-  describe('verifyComponent', () => {
+  describe(`verifyComponent "${Component.displayName}"`, () => {
```

Comment:
This helps identify multiple `verifyComponents` when it comes to compound components